vterm/stdout.go

```diff
@@ -99,9 +99,25 @@ func (v *VTerm) ProcessStdout(input *bufio.Reader) {
 				}
 
 				v.RedrawWindow() // FIXME inefficient
-			case ecma48.DCH: // delete characters
+			case ecma48.ECH: // erase characters - makes characters to the right blank
 				if x.N > v.w-v.Cursor.X {
-					x.N = v.w - v.Cursor.X // FIXME: verify that we don't need +/- 1
+					x.N = v.w - v.Cursor.X - 1
+				}
+				for i := 0; i < x.N; i++ {
+					x := v.Cursor.X + i
+					y := v.Cursor.Y
+					v.Screen[y][x] = ecma48.StyledChar{
+						Rune: ' ', IsWide: false, Style: v.Cursor.Style,
+					}
+					v.renderer.HandleCh(ecma48.PositionedChar{
+						Rune: ' ', IsWide: false, Cursor: ecma48.Cursor{
+							X: v.x + x, Y: v.y + y, Style: v.Cursor.Style,
+						},
+					})
+				}
```

Comment:
Maybe there could be some abstraction used here like [`setChar`](https://github.com/aaronjanse/3mux/blob/master/vterm/ops.go#L163-L173), though the only modification that we would need for setChar is to add a `IsWide` parameter.

Reply:
Oh, that's the function I was looking for! Thank you :-)